lib/active_fedora/aggregation/list_source.rb

```diff
@@ -0,0 +1,77 @@
+module ActiveFedora
+  module Aggregation
+    class ListSource < ActiveFedora::Base
+      property :head, predicate: ::RDF::Vocab::IANA['first'], multiple: false
+      property :tail, predicate: ::RDF::Vocab::IANA.last, multiple: false
+      property :nodes, predicate: ::RDF::DC::hasPart
+
+      def save(*args)
+        return true if has_unpersisted_proxy_for? || !changed?
+        persist_ordered_self if ordered_self.changed?
+        super
+      end
+
+      def changed?
+        super || ordered_self.changed?
+      end
+
+      # Ordered list representation of proxies in graph.
+      def ordered_self
+        @ordered_self ||= ordered_list_factory.new(resource, head_subject, tail_subject)
+      end
+
+      # Allow this to be set so that -=, += will work.
+      # @param [ActiveFedora::Orders::OrderedList] An ordered list object this
+      #   graph should contain.
+      def ordered_self=(new_ordered_self)
+        @ordered_self = new_ordered_self
+      end
+
+      # Serializing head/tail/nodes slows things down CONSIDERABLY, and is not
+      # useful.
+      def serializable_hash(options=nil)
+        options ||= {}
```

Comment:
Why not do this in the method declaration? `def serializable_hash(options = {})`